easybuild/easyconfigs/p/PETSc/PETSc-3.9.1-foss-2018a-downloaded-deps.eb

```diff
@@ -0,0 +1,35 @@
+name = 'PETSc'
+version = '3.9.1'
+versionsuffix = '-downloaded-deps'
+
+homepage = 'http://www.mcs.anl.gov/petsc'
+description = """PETSc, pronounced PET-see (the S is silent), is a suite of data structures and routines for the
+ scalable (parallel) solution of scientific applications modeled by partial differential equations."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+toolchainopts = {'usempi': True, 'pic': True}
+
+source_urls = ['http://ftp.mcs.anl.gov/pub/petsc/release-snapshots']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['742e838a35d278693e956ed1ca4592c1d663451f6beea0694bf334aeb67681e8']
+
+# (build) dependencies provided through EasyBuild
+builddependencies = [('CMake', '3.11.1')]
+dependencies = [
+    ('HDF5', '1.10.1'),
+    ('Valgrind', '3.13.0'),
```

Comment:
Maybe explicitly add `--with-debugging` as a configure option since we're including `Valgrind` as a dependency (it's enabled by default according to the PETSc documentation, but it doesn't hurt making it explicit, cfr. --with-debugging).